Frequently-Asked-Questions.mediawiki

```diff
@@ -42,15 +42,15 @@ If not, please file a request at your GNU/Linux distribution's issue tracker.
 
 == Is it legal to use qBittorrent? ==
 qBittorrent is a peer to peer (P2P) file sharing software.<br />
-Although the software is '''perfectly legal''', it may be illegal to download restricted content with this software, depending on the law in your country.
+Although the software is '''perfectly legal''', it may be illegal to download restricted content with this software, depending on the laws in your country.
```

Comment:
I believe "law" is an appropriate term as it is used in the meaning of "legislation", IIRC.

Reply:
This may be a regionalism.  FWIW, to my Canadian/US ear "laws in your country" just _sounds right_ , but I don't have a style guide reference to point to.  Per google the "laws" version appears in 3.9MM pages, and the "law" version appears in 900k.  Can redact.